src/vs/editor/contrib/quickFix/browser/quickFixCommands.ts

```diff
@@ -86,22 +110,51 @@ export class QuickFixController implements IEditorContribution {
 	}
 
 	private _handleLightBulbSelect(coords: { x: number, y: number }): void {
-		this._quickFixContextMenu.show(this._lightBulbWidget.model.fixes, coords);
+		this.quickFixContextMenu.show(this.lightBulbWidget.model.fixes, coords);
 	}
 
 	public triggerFromEditorSelection(): void {
-		this._model.trigger('manual');
+		if (this._model) {
```

Comment:
Yeah, don't do this. Keep the model, enable/disable the lightbulb. The manual invocation should always work